<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>高校挺身式跳远评测系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/logo.png</normaloff>resources/logo.png</iconset>
  </property>
  <widget class="QPushButton" name="RealTimeBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>实时检测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectVideoBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>选择视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SettingBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>20</width>
     <height>500</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
